shared/macros-oval.jinja

```diff
@@ -173,12 +174,21 @@
   This macro can take two parameters:
     - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
     - multi_value (boolean): If set, it means that the parameter can accept multiple values and the expected value must be present in the current list of values.
+    - quotes (String): If non-empty, one level of matching quotes is considered when checking the value. Specify one or more quote types as a string.
+      For example, for shell quoting, specify quotes="'\""), which will make sure that value, 'value' and "value" are matched, but 'value" or '"value"' won't be.
 #}}
-{{%- macro oval_line_in_file_state(value='', multi_value='') -%}}
+{{%- macro oval_line_in_file_state(value='', multi_value='', quotes='') -%}}
```

Comment:
There is an almost identical macro that has the same name defined in the crypto policy check (`harden_ssh_client_crypto_policy`). That macro allows to instantiate multiple states in the OVAL def, as it uses the supplied parameter (that wouldn't be needed otherwise) as part of the state ID.
Unifying those two macros would be outside of scope of this PR.

Reply:
That's unfortunate conflict, we probably should rename the custom macro.